### PR TITLE
Switch to using containers/common pkg's

### PR DESCRIPTION
**Pull request by rhatdan**
pkg/secrets has been moved and slightly been rewrittin
in containers/common, along with pkg/umask.

Convert Buildah to use these packages rather then internal
packages.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
@nalind @giuseppe @ashley-cui @QiWang19 @umohnani8 PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

File: run_linux.go [hunk 1]
```diff
@@ -477,15 +477,15 @@ func (b *Builder) setupMounts(mountPoint string, spec *specs.Spec, bundlePath st
 		return errors.Wrapf(err, "error determining work directory for container %q", b.ContainerID)
 	}
 
-	// Figure out which UID and GID to tell the secrets package to use
+	// Figure out which UID and GID to tell the subscritions package to use
```
Comment (nalind) [inline run_linux.go:480]: Typo: subscriptions.

**Pull request merged**